app/controllers/departments_controller.rb

```diff
@@ -1,9 +1,11 @@
 class DepartmentsController < ApplicationController
-  before_action :set_department, only: [:show, :edit, :update, :destroy]
   include MakeNotice
 
+  before_action :set_department, only: [:show, :edit, :update, :destroy]
+
   def index
-    @departments = policy_scope(Department)
+    authorize Department
+    @departments = DepartmentPolicy::Scope.new(current_user, Department).resolve_superadmin_or_owner
```

Comment:
Again, I think `policy_scope(Department)` should do the same as `DepartmentPolicy::Scope.new(current_user, Department)`?